Add T-lettered article stubs (#4698)
ppy/osu-wiki: wiki/Beatmaps/TAG/en.md

---
stub: true
tags:
  - multiplayer
  - co-op
---

# TAG

> **TAG** is a term used to describe a type of [beatmap](/wiki/Beatmaps) that is designed for more than one person to play at the same time in [multiplayer](/wiki/multi) mode, using the [Tag Coop or Tag Team Vs](/wiki/Multi#tag-coop-/-tag-team-vs) mode.

Reviewer: ```suggestion
**TAG** is a term used to describe a [multiplayer](/wiki/multi) game mode named [Tag Coop or Tag Team Vs](/wiki/Multi#tag-coop-/-tag-team-vs), where players try to complete a [beatmap](/wiki/Beatmaps) cooperatively. Beatmaps that are specifically designed for this mode are usually referred to as **TAG beatmaps/difficulties**.
```
I felt the need to rewrite this line since TAG diffs are a thing because of that very multiplayer mode

> In these modes, each player is responsible for one [comboset](/wiki/Beatmapping/Combo), and throughout the song both players switch who plays between each comboset. Because of this, TAG maps are designed in such a way that their combosets are mapped much farther apart than usual, making it nearly impossible to play by a single player.

Reviewer: ```suggestion
In these modes, each player is responsible for one [comboset](/wiki/Beatmapping/Combo). Throughout the song, players switch between each comboset. For this reason, TAG beatmaps are designed in such a way that their combosets are mapped much farther apart and more complex than the usual, making it nearly impossible to play by a single player.
```

TAG [difficulties](/wiki/Difficulties) are usually denoted by the use of the term "TAG" in the beatmap difficulty name. Most often "TAG" will be followed by a number, which typically denotes how many players the beatmap is meant for (e.g., "TAG2 for two players, TAG4 for four players, etc.).

> Due to their nature, TAG difficulties are always unranked because they are intentionally designed to be near impossible for a single player, although this has not stopped users from trying.

Reviewer 2: ```suggestion
Due to their nature, TAG difficulties are either unranked or Loved, because they are intentionally designed to be near-impossible for a single player.
```
